src/utils/components/AppText.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Text } from 'react-native'
+
```

Comment:
Smart work moving the default Text component into a custom component. I think it might help to export one component and then pass in a prop (probably called `variant`) that contains the particular variant of the font.
e.g.

```javascript
   <CustomText variant="bold" />
   <CustomText variant="regular" />
```
The implement is up to you.
Also, if its not too stressful you can add `propTypes` or `flow` for the component and make `style` a `TextStyle` from react native.
e.g.
```
  PoppinsBold. propTypes = {
   style: TextStyle
 }
```
Reason: You loose the autocomplete functionality that the default Text component provides wihout adding this...

Reply:
On it